plugins/de.cau.cs.kieler.klighd.piccolo.batik/.classpath

```diff
@@ -11,16 +11,12 @@
 	<classpathentry exported="true" kind="lib" path="lib/lib/js.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/lib/pdf-transcoder.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/lib/xalan-2.6.0.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/lib/xerces_2_5_0.jar"/>
```

Comment:
I am not sure if the changes from this commit removes anything needed for this piccolo.batik plugin to work properly. Have you tested this export feature before removing  these dependencies?